CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [unreleased]
  ### Added
- ### Changed 
+* Feature toggle for ehrscape API (ehrbase.rest.ehrscape.enabled, default: false) ([#1415](https://github.com/ehrbase/ehrbase/pull/1415))
```

Comment:
should we note that this is a "breaking" change because prior the API was deprecated but accessable?

Reply:
@vmueller-vg I've added some info in [CDR-1609 Add ehrscape hint to UPDATING.md](https://github.com/ehrbase/ehrbase/pull/1415/commits/34037e2b3e8f9528481bf673f7c2d53d815ee503)